docs/eventing/sources/apiserversource/getting-started/_index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Getting started"
+weight: 01
+type: "docs"
+showlandingtoc: "false"
+aliases:
+  - /docs/eventing/samples/kubernetes-event-source
+---
+
+## Prerequisites
+
+Before you can create an API server source, you must install Knative Eventing and the `kubectl` CLI tool.
+
+## Create an API server source
+
+1. Optional: Create a namespace for the API server source instance:
+
+    ```shell
+    kubectl create namespace <new_namespace_name>
+    ```
+
+    Creating a namespace for your API server source and related components allows you to view changes and events for this workflow more easily, since these are isolated from the many other components that may exist in your `default` namespace.
+
+    It also makes removing the source easier, since you can simply delete the namespace to remove all of the resources.
+
+1. Create a service account:
+
+    ```yaml
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: <service_account_name>
+      namespace: <your_namespace>
+    ```
+
+1. Create a cluster role:
+
+    ```yaml
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: <cluster_role_name>
+    rules:
+    - apiGroups:
+      - ""
+      resources:
+      - events
+      verbs:
+      - get
+      - list
+      - watch
+    ```
+
+1. Create a cluster role binding:
+
+    ```yaml
```

Comment:
For yaml, I think we should standardize on this style:

```shell
kubectl create -f - <<EOF
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
    name: <cluster_role_binding_name>
  roleRef:
    apiGroup: rbac.authorization.k8s.io
    kind: ClusterRole
    name: <your_cluster_role>
  subjects:
  - kind: ServiceAccount
    name: <your_service_account>
    namespace: <your_namespace>
EOF
```

What do you think?

Then the tabs are `kubectl` or `kn`

Reply:
If the rest of the YAML on the site looks like what you posted Ashleigh, I would keep it. 

I do, however, agree that we should tell people how/when to run the files we have them create?

Reply:
It might be worth rather than making the whole thing copy/paste, giving the `kubectl apply` commands and the YAML as proper steps and add the variables and descriptions for the YAML/commands. We can talk about it in person tomorrow.

Reply:
lol @omerbensaadon we didn't talk about any standardization stuff on our call like I thought we would; do we want to create a proposal re the style guide stuff too? Can't remember if we decided on that after the long weekend.